packages/frontend/web/pages/Article.tsx

```diff
@@ -55,6 +58,11 @@ export const Article: React.FC<{
         />
         <main className={articleWrapper}>
             <Container className={articleContainer}>
+                <Island
+                    c="ClientComponent"
+                    s={data => <ClientComponent />}
```

Comment:
```suggestion
                    s={ClientComponent}
```

otherwise it'll get initialised without props on it